gulpfile.js

```diff
@@ -180,11 +181,18 @@ gulp.task('docs:mkdocs', shell.task([
   'mkdocs serve'
 ], {
   cwd: 'docs'
-}))
+}));
+
+gulp.task('lint', function () {
+  return gulp.src(['src/pathfora.js', 'gulpfile.js', 'test/pathforaSpec.js', 'docs/docs/examples/**/*.js'])
+    .pipe(eslint())
+    .pipe(eslint.format())
+    .pipe(eslint.failAfterError());
+});
 
 gulp.task('test', ['build:styles', 'build:testjs']);
 gulp.task('build:local', ['build:styles', 'local:js']);
```

Comment:
I actually originally had it in all of the build tasks, and it was a PAIN while developing later, so I decided to take it out. Maybe we need some task to run before you push up a commit? That way we're not always linting and it doesn't slow down the dev process but we ensure people actually lint before commiting? Idk, you should try it with the linting in the build task, and tell me if it's annoying to you.